tool/README.md

```diff
@@ -28,24 +27,56 @@ Running each tool without any options (e.g. `./awslc_bm`) will run all available
 Additionally, there are a number of arguments that enable different functionality:
 * `-filter` provides a filter on the benchmarking tests to be run.
 * `-timeout` sets the number of seconds each test is run for (default 1).
+* `-threads` is a comma-separated list of thread counts to run multithreaded benchmarks (default is "1,2,4,8,16,32,64")
 * `-chunks` is a comma-separated list of input sizes to run tests at (default is "
   "16,256,1350,8192,16384)
 * `-json` has the tool print the output of each benchmark in JSON format.
 
 For example, `./awslc_bm -filter AES -timeout 10 -chunks 16, 256, -json` will run all AES-related tests with input sizes of 16 and 256 for 10 seconds, and output the results in JSON format.
 
-## Setup
-In order to build the above-mentioned benchmarking tools, absolute paths to each libaries' install location must be provided via compiler flags.
-
-### Compiler Flags
-|  Tool Name  |  Compiler Flag  |
-| ------------- | ------------- |
- | bssl speed | (none) | 
-| awslc_bm | -DAWSLC_INSTALL_DIR |
-| bssl_bm | -DBORINGSSL_INSTALL_DIR |
-| ossl_1_0_bm | -DOPENSSL_1_0_INSTALL_DIR |
-| ossl_1_1_bm | -DOPENSSL_1_1_INSTALL_DIR |
-| ossl_3_0_bm | -DOPENSSL_3_0_INSTALL_DIR |
+## Comparison Setup
+The AWS-LC benchmark supports building and running with other common libcryptos.
+Build and install the other libcryptos you would like to test with locally, for 
+example building AWS-LC 2022 FIPS branch and OpenSSL 3.3 to compare with AWS-LC
+main branch:
+
+```bash
+git clone -b openssl-3.3 --depth 1 https://github.com/openssl/openssl.git ~/openssl-3.3-src
+(cd ~/openssl-3.3-src && ./config --prefix="${HOME}/openssl-3.3-install" --openssldir="${HOME}/openssl-3.3-install" && make -j && make install_sw)
```

Comment:
Why the parentheses around the build/install commands?

Reply:
It executes the commands in a subshell and doesn't affect the user's current working directory, I'll tweak this to use pusd/popd.